### ISSUE TITLE
Android 12 targetSdkversion 30 shows  java.lang.NullPointerException: Attempt to invoke interface method 'java.lang.String io.liteglue.SQLDatabaseHandle.getLastErrorMessage()' on a null object reference

### ISSUE THREAD

**Issue by nadirabanu** [brody2consult/cordova-sqlite-storage#995]
we have updated Android sdkversions as below and have an issue when we run it on android 12. Any help is appreciated.
     defaultTargetSdkVersion= 30
      defaultCompileSdkVersion=30 

java.lang.NullPointerException: Attempt to invoke interface method 'java.lang.String io.liteglue.SQLDatabaseHandle.getLastErrorMessage()' on a null object reference
        at io.liteglue.SQLiteGlueConnection.<init>(SQLiteGlueConnection.java:12)

**Comment by bilalsaeed** [comment]
Hi @nadirabanu 

did you manage to resolve it?

**Comment by nadirabanu** [comment]
yes @bilalsaeed  I followed the recommendation given below and mentioned in the link provided below. Now it works fine on all devices.

 The this.oldImpl = true is the same result of using the mentioned workaround androidDatabaseProvider: 'system'.

https://github.com/OutSystems/Cordova-sqlite-storage/pull/7

**Comment by bilalsaeed** [comment]
I upgraded the plugin on latest and it worked.
